documentation/directory_structure.md

```diff
@@ -59,3 +61,49 @@ Follow these tips for managing standalone modules:
 -   Add `modules/*` to `.gitignore` of your project to prevent accidentally committing standalone modules.
 -   When you run tasks and plans within a project directory the modulepath is searched in order for modules containing Bolt content. The Bolt project directory itself is loaded as a module at the front of the modulepath, and the default modulepath is `<project directory>/modules:<project directory>/site-modules`. If you have a module in both the `modules` and `site-modules` directories, the version in `modules` will be used.
 -   As a best practice, write automated tests for the tasks and plans in your module, if possible. For information about automated testing patterns, check out these resources: [Example of unit testing plans and integration \(acceptance\) testing tasks](https://github.com/puppetlabs/puppetlabs-facts) (GitHub) and [Writing Robust Puppet Bolt Tasks: A Guide](https://puppet.com/blog/writing-robust-puppet-bolt-tasks-guide) (Puppet blog).
+
+## Simplified directory structure
+
+If you're working on a new project and you want to use a simplified directory
+structure, you can load your plan and task content directly from your project as
+if it was a module. 
+
+> **Note:** This is an experimental feature.
+
+To load your content from a Bolt project:
+1. Create a `project.yaml` file in the root directory of your Bolt project. 
+2. Place your Bolt plans and tasks in `plans` and `tasks` directories
+   respectively.
+
+If `project.yaml` exists at the root of a project directory, and your tasks and
+plans are located in their respective directories, Bolt loads the project as a 
+module namespaced to the name of the directory.
```

Comment:
```suggestion
If `project.yaml` exists at the root of a project directory, Bolt loads the project as a module.
Tasks and plans are loaded from the `tasks/` and `plans/` directories and are namespaced
to the project name.
```

You don't need any tasks or plans for Bolt to load the project as a module. If the directories are missing or don't have any content, the project is loaded like any other module that doesn't have tasks or plans.